controllers/status-check-override.js

```diff
@@ -0,0 +1,73 @@
+var scOverride = require('status-check');
+
+delete scOverride['checkSingleLink'];
+
+scOverride.checkSingleLink = function (linksArray, index, callBack, outputArray, showProgressInConsole) {
+  var presentObject = this,
+    http = require('http'),
+    https = require('https'),
+    method = https,
+    alterFlag = 0,
+    originalLink = "";
+  if (linksArray[index].indexOf('https://') === -1 && linksArray[index].indexOf('http://') === -1) {
+    originalLink = linksArray[index];
+    linksArray[index] = "http://" + linksArray[index];
+    alterFlag = 1;
+  }
+  if (linksArray[index].indexOf('https') == -1) {
```

Comment:
I think it would be more readable/make more sense to test if the method is http instead of testing if it is not https.

Reply:
The thing is: if I have a link like "https://raven-os.org/" and I check for "http", I will find it. As "http" is in "https". So it's better to check if it's not https.

Reply:
Check for `https://` then =)